src/channels.ts

```diff
@@ -730,9 +731,9 @@ export default class Channels {
     /**
      * @method AddUserGroup - Adds user group to channel when provided with a valid token,
      * channel id and a user group id.
-     * @param {string []} userGroupIds - User Group IDs
```

Comment:
It does take in a string array. Revert the changes you made here and for `removeUserGroup`